main.go

```diff
@@ -26,7 +26,7 @@ type Rsync struct {
 	Target    string   `json:"target"`
 	Delete    bool     `json:"delete"`
 	Recursive bool     `json:"recursive"`
-	Exclude   string   `json:"exclude"`
+	Exclude   []string `json:"exclude"`
```

Comment:
Huh, sry, I'm a real golang n00b who got just a few chapters behind hello world ;) but I'll investigate & try something new ;) thx for the tip